Dockerfile.ocp

```diff
@@ -1,15 +1,14 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.11 AS builder
 WORKDIR /go/src/github.com/prometheus/prometheus
 COPY . .
-ARG BUILD_PROMU=false
-RUN yum install -y prometheus-promu && make build && yum clean all
+RUN if yum install -y prometheus-promu; then export BUILD_PROMU=false; fi && make build 
```

Comment:
`yum clean all` is still needed somewhere

Reply:
We don't need to clean yum cache as this is a multi-stage build and installation is invoked in temporary builder container. Yum cache will be dropped either way and we are only copying binaries to next image.

Reply:
Ah right!